serializer.js

```diff
@@ -310,6 +319,49 @@ function _buildLiteralObject(value, metaProperty)
   return literal;
 }
 
+function _collectActions(trailEntity, graph, options)
+{
+    var graphName = options.defaultGraph || null;
+
+    if(trailEntity.parent)
+    {
+        graphName = trailEntity.parent;
+    }
+    
+    if(trailEntity.actions && trailEntity.actions.length>0)
+    {
+        //does not have effect in triplestore injection
```

Comment:
what is the intended meaning of triplestore injection here? something like SQL injection?